homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -77,6 +82,7 @@
 MODEL_AIRPURIFIER_SA1 = "zhimi.airpurifier.sa1"
 MODEL_AIRPURIFIER_SA2 = "zhimi.airpurifier.sa2"
 MODEL_AIRPURIFIER_2S = "zhimi.airpurifier.mc1"
+MODEL_AIRPURIFIER_3 = "zhimi.airpurifier.ma4"
```

Comment:
Please also add MODEL_AIRPURIFIER_3H = "zhimi.airpurifier.mb3" with similar attributes and features as MODEL_AIRPURIFIER_3